docs/user-docs/latest/users-roles-permissions/configuring-end-users-roles.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Configure End-users Roles - Strapi User Guide
+description: Instructions to configure the end-users roles for a front-end application using the Users & Permissions plugin
+---
+
+# Configuring end-users roles
+
+End-users are the users who consume the content that is created and managed with a Strapi application and displayed on front-end applications (e.g. websites, mobile applications, connected devices etc.). Unlike the administrators, they do not have access to the admin panel. End-users are managed with the Users & Permissions plugin (see [Users & Permissions plugin](../plugins/strapi-plugins.md#users-permissions-plugin)). This plugin is however not entirely managed and configured from one same place of the admin panel: end-users accounts are managed from the Content Manager (see [Managing end-user accounts](../users-roles-permissions/managing-end-users.md)) but end-users roles and permissions are managed in the Settings interface.
```

Comment:
Really great definition, Mégane! 👏 

By the way, I think the end of this paragraph highlights the fact that configuring users, roles & permissions is a bit tricky with Strapi 😅 This will probably change for the better with v4 😄